packages/core/src/browser/style/tabs.css

```diff
@@ -269,7 +269,7 @@
   display: flex;
   align-items: center;
   margin-left: 8px; /* `padding` + `margin-right` from the container toolbar */
-  opacity: 0.25;
+  opacity: 0.4;
```

Comment:
maybe we can have a css variable for unavailable opacity and use it instead across css files?